tempesta_fw/http.c

```diff
@@ -849,6 +849,15 @@ tfw_h2_send_resp(TfwHttpReq *req, int status, unsigned int stream_id)
 	mit = &resp->mit;
 	skb_head = &resp->msg.skb_head;
 	body = TFW_STR_BODY_CH(msg);
+	if (WARN_ON_ONCE(body->len > ctx->rsettings.max_frame_sz)) {
+		/*
+		 * TODO: split body to frames it it's too long.
+		 * Since full-featured response is not constructed construct
+		 * here, it's possible to avoid framing body for each h2 client
+		 * individually. Every client must support 16384-byte frames.
+		 */
```

Comment:
Please add issue numbers for TODO - it's handy to see when exactly and in context of which issues a particular TODO must be done